src/playback-watcher.js

```diff
@@ -154,18 +154,30 @@ export default class PlaybackWatcher {
    * @private
    */
   fixesBadSeeks_() {
-    let seekable = this.seekable();
-    let currentTime = this.tech_.currentTime();
+    const seeking = this.tech_.seeking();
+    const seekable = this.seekable();
+    const seekableStart = seekable.start(0);
```

Comment:
`seekable` can have a length of 0, in which case this will error. Both `beforeSeekableWindow_` and `afterSeekableWindow_` check the length and return false if it is 0, so you could not create these const here and just access `seekable.start(0)` if you get into the appropriate if block. Otherwise you will have to check the length here as well

Reply:
I'll move these down to their appropriate if blocks